test/maps/azalias/zz_generated_models.go

```diff
@@ -137,7 +137,7 @@ type GeoJSONFeatureData struct {
 // GeoJSONObjectClassification provides polymorphic access to related types.
 // Call the interface's GetGeoJSONObject() method to access the common type.
 // Use a type switch to determine the concrete type.  The possible types are:
-// - *GeoJSONObject, *GeoJsonFeature
+// - *GeoJSONFeature, *GeoJSONObject
```

Comment:
For discriminated types, we were generating the comment before the type names were fixed up.  So this is an added bonus.

Reply:
I further refined this fix to include names that were changed due to stuttering.